src/Client.php

```diff
@@ -311,6 +316,47 @@ private function prepareEvent(Event $event, ?EventHint $hint = null, ?Scope $sco
         return $event;
     }
 
+    private function applyIgnoreOptions(Event $event, ?EventHint $hint): ?Event
+    {
+        if ($event->getType() === EventType::event()) {
+            $exceptions = $event->getExceptions();
+
+            if (empty($exceptions)) {
+                return $event;
+            }
+
+            foreach ($exceptions as $exception) {
+                if (\in_array($exception->getType(), $this->options->getIgnoreExceptions(), true)) {
```

Comment:
The only thing is that this works slight different from the now deprecated `IgnoreErrorsIntegration` because that uses `is_a`, not sure if intentional but it might be nice to say you want to ignore `HttpException::class` instead of `[ClientHttpException::class, ServerHttpException::class, ...]` on the other hand this is more "expected" and you can still make that logic work with `before_send` of course.

Just a little rambling, I have no hard feelings either way so fine with the current impl.